### PR TITLE
Onboarding skip enable notifications screen

### LINKED ISSUE #21627: Onboarding - Remove Enable notifications screen
Link to figma: https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY?node-id=15071-95173#1025688546

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/21627

### Video

https://github.com/user-attachments/assets/b253460d-955a-45b5-acca-99d14a883924

### Testing

- The PR doesn’t require manual testing, as the change is minimal. (However, end-to-end tests may need adjustments.)

status: ready

### CODE REVIEW

File: src/status_im/contexts/onboarding/enable_notifications/view.cljs [hunk 1]
```diff
@@ -19,26 +17,21 @@
     :description                     (i18n/label :t/enable-notifications-sub-title)
     :description-accessibility-label :notifications-sub-title}])
 
-(defn- finish-onboarding
-  [notifications-enabled?]
-  (rf/dispatch [:push-notifications/switch notifications-enabled?])
-  (shell.utils/change-selected-stack-id shell.constants/default-selected-stack true nil)
```
Comment (Parveshdhull) [inline src/status_im/contexts/onboarding/enable_notifications/view.cljs]: `change-selected-stack-id` is not needed anymore - https://github.com/status-im/status-mobile/pull/21634

Comment (Parveshdhull) [reply src/status_im/contexts/onboarding/enable_notifications/view.cljs]: Marked the PR as draft, as it should be merged after: https://github.com/status-im/status-mobile/pull/21634

Comment (Parveshdhull) [reply src/status_im/contexts/onboarding/enable_notifications/view.cljs]: ready for review.

**Review by qfrank — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by Parveshdhull** [comment]
@status-im/mobile-qa Could you please check if the e2e tests need any adjustments?

**Comment by pavloburykh** [comment]
@Parveshdhull thanks for the PR! @yevh-berdnyk could you please adjust e2e tests according to current changes? Thank you!

**Comment by Parveshdhull** [comment]
@yevh-berdnyk, Thank you for fixing e2e tests. PR ready to merge?
cc @pavloburykh

**Comment by yevh-berdnyk** [comment]
> @yevh-berdnyk, Thank you for fixing e2e tests. PR ready to merge? cc @pavloburykh

Hi @Parveshdhull, thanks for the PR, it's ready now. E2E test failure is not related

**Pull request merged**